### DISCUSSION TITLE
Some beatmaps dont want to update

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19306] [Q&A] [answered]
https://user-images.githubusercontent.com/86934170/180592577-569d9859-ef1c-442f-bf35-1bb29e7a8322.mp4

Even after restart or changing beatmap in editor.
I guess it happens because of beatmap's changed state. (Qualified -> Ranked)

https://user-images.githubusercontent.com/86934170/180592849-fdbb9b82-6bdf-41d9-bfd1-97d48f47898a.mp4

[network.log](https://github.com/ppy/osu/files/9173185/network.log)
[runtime.log](https://github.com/ppy/osu/files/9173186/runtime.log)
[database.log](https://github.com/ppy/osu/files/9173187/database.log)
[performance.log](https://github.com/ppy/osu/files/9173188/performance.log)
[updater.log](https://github.com/ppy/osu/files/9173189/updater.log)

**Comment by Theighlin** [comment]
I'm not sure it's because the rank status changed. I too have a qualified beatmap that didn't update to ranked, but unlike yours, it doesn't prompt me to update it infinitely.

**Comment by peppy** [comment]
I can reproduce this. It's due to an issue I knew existed (the local `online.db` is outdated).

To try fixing, you can delete `online.db` from your osu! folder then run the game again.

**Reply by Loreos7** [reply]
Doesn't work

**Reply by Loreos7** [reply]
Its just doing the same but the fun fact game shows actual leaderboard from the beatmap (Ranked status) but locally its still Qualified

**Comment by peppy** [comment, marked as answer]
Should be resolved via #19440.

**Reply by Loreos7** [reply]
Nothing has changed with 731.1

https://user-images.githubusercontent.com/86934170/182003188-19c828bd-82f0-434c-8d97-366997fbbfb8.mp4

**Reply by Loreos7** [reply]
Other ones dont even show the update button (but they are all already ranked):
![image](https://user-images.githubusercontent.com/86934170/182003340-65f923e4-50c0-4d81-847e-1307f97ceefd.png)
![image](https://user-images.githubusercontent.com/86934170/182003350-e77afcea-8159-4be2-8819-d4fd114d6a8c.png)
![image](https://user-images.githubusercontent.com/86934170/182003358-1a1165d4-b923-4dc2-ad00-921e756139b5.png)

**Reply by Loreos7** [reply]
See what happens if I edit a beatmap in editor

https://user-images.githubusercontent.com/86934170/182003431-902516fa-2ec9-43e4-86e3-83b2a5c93231.mp4

**Reply by peppy** [reply]
The video doesn't load for me. The screenshots aren't really saying much. They won't show update because they are in the correct online state. They won't change from "qualified" to "ranked" because there's no mechanism to handle that yet.

**Reply by frenzibyte** [reply]
> See what happens if I edit a beatmap in editor
> 
>  2022-07-31.02-25-39.mp4

That video you just attached shows a completely different issue (editing a beatmap does not invalidate status) from any of the above (beatmaps don't update from qualified to ranked). Please avoid mixing different issues in one thread, open a separate one.

**Reply by peppy** [reply]
Local changes will be better displayed via #19529.

**Comment by Feodor0090** [comment]
Right after download:
![изображение](https://user-images.githubusercontent.com/53872073/184871786-41935132-dff0-443b-8550-0d7e660c736c.png)
https://osu.ppy.sh/beatmapsets/3681#osu/23326

**Reply by Loreos7** [reply]
This relates to https://github.com/ppy/osu/discussions/19551

**Reply by frenzibyte** [reply]
Please include any other beatmap you encounter in #19551.
